src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/ResourceOwner.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Uno.UI.SourceGenerators.XamlGenerator
+{
+	/// <summary>
+	/// Resource owner variable definition
```

Comment:
Would be nice if you can expand this - it's not very obvious why we need Yet Another notion of scope.

Reply:
Will do in a subsequent PR